src/aws.ts

```diff
@@ -422,18 +423,18 @@ export namespace $AWS {
         } else if (input.kind !== "ObjectLiteralExpr") {
           throw new Error("argument 'input' must be an ObjectLiteralExpr");
         }
-        const functionName = input.getProperty("FunctionName")?.expr;
+        const functionName = input.getProperty("Function")?.expr;
         if (functionName === undefined) {
-          throw new Error("missing required property 'FunctionName'");
+          throw new Error("missing required property 'Function'");
         } else if (functionName.kind !== "ReferenceExpr") {
           throw new Error(
-            "property 'FunctionName' must reference a functionless.Function"
+            "property 'Function' must reference a functionless.Function"
           );
         }
         const functionRef = functionName.ref();
         if (!isFunction(functionRef)) {
           throw new Error(
-            "property 'FunctionName' must reference a functionless.Function"
+            "property 'Function' must reference a functionless.Function"
```

Comment:
Did we somehow miss these or is this a merge problem?

Reply:
Not sure! This is why tests that actually execute the code is important.

Reply:
I think it was missed, the last change on that line was 2 months ago.